src/lib/qnio/utils.c

```diff
@@ -9,6 +9,117 @@
  */
 
 #include "defs.h"
+#include "qnio.h"
+
+int 
+is_secure()
+{
+    if (access(SECURE_IMPL, F_OK) != 0)
```

Comment:
How would this work if this was driven by QEMU? I assume somewhere on the host there would be a VxHS configuration file which would indicate whether to use SSL or not. This would be set to what the VxHS user has chosen. Would it be better if libvxhs uses a configuration file say /etc/vxhs.conf for this?
Also if the choice is driven by the VxHS user and the implementation is in libvxhs then why does QEMU have to have control over this?